ui/main/src/app/services/authentication/authentication.service.ts

```diff
@@ -154,10 +154,12 @@ export class AuthenticationService {
      * @return true if the expiration date stored in the `localestorage` is still running, false otherwise.
      */
     verifyExpirationDate(): boolean {
+        const secondsToCloseSession = this.configService.getConfigValue('secondsToCloseSession', 60);
```

Comment:
I would prefer having this const global and initialize in   assignConfigurationProperties()